### DISCUSSION TITLE
Print BTC vouchers

### DISCUSSION THREAD

**Discussion by fossadmin** [btcpayserver/btcpayserver#4021] [Ideas / Feature Requests]
I was wondering if it's possible to print out bitcoin vouchers to sell similar to azte.co? Might be super useful for business that accept BTC but need to cashout for fiat without technically "selling BTC" since we'd be selling a voucher that the customer uses to redeem BTC. I think this would help with circular economy by giving customers both options to spend BTC and buy BTC (no KYC) at businesses that offer this.

**Comment by Kukks** [comment]
This is already possible. 
* Set up btcpay
* Set up store
* Create a pull payment with amount of voucher
* Design a voucher and add link of pull payment to voucher
* BTC Vouchers!
